frontend/src/components/GTimeTextField.vue

```diff
@@ -76,16 +72,11 @@ const inputHint = computed(() => {
 })
 
 const errorMessages = computed(() => {
-  return [
-    ...getErrorMessages('timeValue'),
-    ...props.errorMessages,
-  ]
+  const errors = getErrorMessages(v$.value.timeValue)
```

Comment:
Does this function return errors (error objects) or error messages. If it returns errorMessages the constant should have the name errorMessages.